packages/react-native-codegen/src/parsers/parser.js

```diff
@@ -12,6 +12,12 @@
 
 import type {ParserType} from './errors';
 
+export interface ParserOptions {
+  enums?: boolean;
+  sourceType?: string;
+  plugins?: Array<string>;
+}
+
```

Comment:
This one will be deleted in an other commit. Sorry for the confusion but I did that 1 week ago and the parser evolved after